packages/@ourworldindata/types/src/dbTypes/PostsGdocs.ts

```diff
@@ -61,5 +63,6 @@ export function serializePostsGdocsRow(row: DbEnrichedPostGdoc): DbRawPostGdoc {
         ...row,
         content: serializePostGdocContent(row.content),
         breadcrumbs: serializePostsGdocsBreadcrumbs(row.breadcrumbs),
+        published: row.published ? 1 : 0,
```

Comment:
jsyk you can also do `Number(row.published)` 🙂 

Again, the ternary is probably more explicit but I just like sharing little JS special recipes